README.md

```diff
@@ -362,6 +363,38 @@ Alternatively, you can also update the service `awx-service` in your namespace t
 #> minikube service <serviceName> -n <namespaceName> --url
 ```
 
+### Generating a bundle
+
+If one has the Operator Lifecycle Manager (OLM) installed, the following steps is the process to generate the bundle that would nicely display in the OLM interface.
+
+At the root of this directory:
+
+1. Build and publish the operator: `operator-sdk build <registry>/<namespace>/<operator-container>:<tag> && podman push <registry>/<namespace>/<operator-container>:<tag>`
+
+2. Build and publish the bundle: `podman build -f bundle.Dockerfile -t <registry>/<namespace>/<bundle-container>:<tag> . && podman push <registry>/<namespace>/<bundle-container>:<tag>`
```

Comment:
nit: I would change the argument order in the `podman build` command as I can see people forgetting to add the `.` on its syntax. Maybe this will cover it better:
```sh
`podman build . -f bundle.Dockerfile -t <registry>/<namespace>/<bundle-container>:<tag>  && podman push <registry>/<namespace>/<bundle-container>:<tag>`
```